scripts/benchmark_inference.py

```diff
@@ -284,12 +291,12 @@ def bench_end_to_end(use_cache, expected):
     print0("Compiling model...")
 
     # This is to prevent a bug in PT 2.1 that has been fixed in PT 2.2 nightlies
-    torch._inductor.config.joint_graph_constant_folding = False
+    # torch._inductor.config.joint_graph_constant_folding = False
     # with mode='reduce-overhead' we see better performance but on multi-GPU models
     # hit an error on the end-to-end test below when run after other tests (if it's
     # run first it works, confirming a memory leak):
     # `RuntimeError: Expected curr_block->ptr == block_state.ptr to be true, but got false.`
-    model = torch.compile(model, dynamic=True, mode=args.compile_mode)
+    model = torch.compile(model, mode=args.compile_mode)
```

Comment:
removing dynamic=True here seems like it should be called out separate from the isort changes